model-optimizer/extensions/front/tf/TFFFTToDFT.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from extensions.ops.dft import DFT, IDFT
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.front.subgraph_matcher import SubgraphMatch
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, Node, rename_nodes
+
+
+class TFFFTToDFT(FrontReplacementSubgraph):
+    """
+    This transformation converts the operation TFFFT into MO DFT (if the attribute 'is_inverse' is False),
+    or into MO IDFT (otherwise).
+    """
+    enabled = True
+
+    def run_after(self):
+        from extensions.front.tf.RollRealImagPackReplacement import RollRealImagPackReplacement
+        return [RollRealImagPackReplacement]
+
+    def pattern(self):
+        return dict(
+            nodes=[('fft', dict(op='TFFFT'))],
+            edges=[]
+        )
+
+    def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
```

Comment:
Avoid usage of patterns when you are looking for a single operation. Instead use the following approach:

```python
    def find_and_replace_pattern(self, graph: Graph):
        for tf_fft in graph.get_op_nodes(type='TFFFT'):

```

Reply:
Fixed.